options.go

```diff
@@ -1,28 +1,30 @@
 package podrick
 
+type Option func(*config)
```

Comment:
Could you please add a doc string to this?
```suggestion
// Option configures the configuration of the started container.
type Option func(*config)
```